qemu/tests/cfg/multi_vms_nic.cfg

```diff
@@ -13,7 +13,7 @@
     ping_counts = 10
     interface_ip_filter = "(eth\d+).*?HWaddr .\w+:\w+:\w+:\w+:\w+:\w+.*?inet addr:(.\d+.\d+.\d+.\d+)"
     Fedora.18, RHEL.7:
-        interface_ip_filter = "(\w+):.*inet (.\d+.\d+.\d+.\d+).*?ether .\w+:\w+:\w+:\w+:\w+:\w+"
+        interface_ip_filter = "(eth\d+).*?inet (.\d+.\d+.\d+.\d+)"
```

Comment:
not all ethernet interface named ethxx, eg, 
[xutian@xutian-dev]$ ifconfig  
eno1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500

more details see this post:
https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Deployment_Guide/appe-Consistent_Network_Device_Naming.html